src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs

```diff
@@ -36,7 +36,7 @@ public class ReturnRequestHandler : IRequestHandler<ReturnRequest>
             _logger = logger;
         }
 
-        public async Task<Unit> Handle(ReturnRequest request, CancellationToken cancellationToken)
+        public Task<Unit> Handle(ReturnRequest request, CancellationToken cancellationToken)
```

Comment:
Guessing changing this from a Task<Unit> to just Unit would be troublesome?

Reply:
@CodeWolf86  Ideally the whole thing should be deleted or refactored as the comment suggests. I didn't wanted to make too many changes... so just took minimal possible route to get rid of the warning.